package.json

```diff
@@ -17,6 +17,9 @@
     "@babel/helper-get-function-arity": "^7.16.7",
     "@datadog/mobile-react-native": "^2.4.3",
     "@datadog/mobile-react-navigation": "^2.4.3",
+    "@gluestack-style/react": "^1.0.57",
+    "@gluestack-ui/config": "^1.1.20",
+    "@gluestack-ui/themed": "^1.1.73",
```

Comment:
Gluestack is now at version 3, version 1 is abandoned. 
I also work on this, you can cherry pick the installation in #2029